nuxt.config.js

```diff
@@ -107,13 +110,12 @@ export default {
         config: {
           /* eslint-disable-next-line no-undef */
           apiKey: process.env.FIREBASE_API_KEY,
-          authDomain: "findadoc-bc230.firebaseapp.com",
-          databaseURL: "https://findadoc-bc230-default-rtdb.firebaseio.com",
-          projectId: "findadoc-bc230",
-          storageBucket: "findadoc-bc230.appspot.com",
-          messagingSenderId: "871643437476",
-          appId: "1:871643437476:web:d7271d01ee125e9f7bc931",
-          measurementId: "G-4PJ823JSZ1",
+          authDomain: process.env.FIREBASE_AUTH_DOMAIN,
```

Comment:
prod variables are moved to netlify secrets.

Reply:
Awesome, I'll have some twitter ones for ya soon too...